src/component/utils/get-edges-from-types/index.ts

```diff
@@ -57,10 +77,10 @@ const buildFieldToEdges =
     ];
   };
 
-const typeToEdges = ({ isShowingEdgeLabels, type }) =>
+const typeToEdges = ({ isShowingEdgeLabels, type }: { isShowingEdgeLabels: boolean; type: any }) =>
   _.map(type.fields, buildFieldToEdges({ fromType: type.name, isShowingEdgeLabels }));
 
-const getEdgesFromTypes = (types, isShowingEdgeLabels) =>
+const getEdgesFromTypes = (types: any, isShowingEdgeLabels: boolean = false): any[] =>
```

Comment:
more refs to type and types around the place, hopefully can just reuse that def in the other file

Reply:
I guess return type is array of string?

Reply:
I've fixed up a lot of the typedefs however the ones where I'm using `types: any` is because there seems to be an incompatibility between `Field` and `Type`. I've raised it with ESD that we need to revisit to address `any` usages and ensure we're using proper typing.